Bottom: 36f7dd8eac4775cafe95f1141884a0bc18e5d9aa
Top:    9b1dafcb79431e0d4a5e4b7c0df3c9b42b6a6a61
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-09-30 22:36:15 +0100

Refresh of draw-pad-bounding-box-to-check

---

diff --git a/src/draw.c b/src/draw.c
index b09af83..38daf9a 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1523,8 +1523,8 @@ DrawPadLowLevel (hidGC gc, PadTypePtr Pad, Boolean clear, Boolean mask)
                       Pad->Point1.X, Pad->Point1.Y,
                       Pad->Point2.X, Pad->Point2.Y);
     }
-  /* Draw bounding box for test */
-  {
+#if 0
+  { /* Draw bounding box for test */
     BoxType *box = &Pad->BoundingBox;
     gui->set_line_width (gc, 1);
     gui->draw_line (gc, box->X1, box->Y1, box->X1, box->Y2);
@@ -1532,6 +1532,7 @@ DrawPadLowLevel (hidGC gc, PadTypePtr Pad, Boolean clear, Boolean mask)
     gui->draw_line (gc, box->X2, box->Y2, box->X2, box->Y1);
     gui->draw_line (gc, box->X2, box->Y1, box->X1, box->Y1);
   }
+#endif
 }
 
 /* ---------------------------------------------------------------------------
